packages/i18nify-go/modules/country_subdivisions/country_subdivisions.go

```diff
@@ -0,0 +1,128 @@
+// This file was generated from JSON Schema using quicktype, do not modify it directly.
+// To parse and unparse this JSON data, add this code to your project and do:
+//
+//    countrySubdivisions, err := UnmarshalCountrySubdivisions(bytes)
+//    bytes, err = countrySubdivisions.Marshal()
+
+// Package country_subdivisions provides functionality to manage subdivisions (such as states, provinces, or regions) and cities within countries.
+package country_subdivisions
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+// DataFile is the directory where JSON files containing country subdivision data are stored.
+const DataFile = "modules/country_subdivisions/"
+
+// UnmarshalCountrySubdivisions parses JSON data into a CountrySubdivisions struct.
+func UnmarshalCountrySubdivisions(data []byte) (CountrySubdivisions, error) {
+	var r CountrySubdivisions
+	err := json.Unmarshal(data, &r)
+	return r, err
+}
+
+// Marshal converts a CountrySubdivisions struct into JSON data.
+func (r *CountrySubdivisions) Marshal() ([]byte, error) {
+	return json.Marshal(r)
+}
+
+// CountrySubdivisions contains information about country subdivisions.
+type CountrySubdivisions struct {
+	CountryName string           `json:"country_name"` // CountryName represents the name of the country.
+	States      map[string]State `json:"states"`       // States contains information about states or provinces within the country.
+}
+
+// GetCountryName returns the name of the country.
+func (r *CountrySubdivisions) GetCountryName() string {
+	return r.CountryName
+}
+
+// GetStates returns information about states or provinces within the country.
+func (r *CountrySubdivisions) GetStates() map[string]State {
+	return r.States
+}
+
+// GetCountrySubdivisions retrieves subdivision information for a specific country code.
+func GetCountrySubdivisions(code string) CountrySubdivisions {
+	// Read JSON data file containing country subdivision information.
+	subDivJsonData, err := ioutil.ReadFile(DataFile + code + ".json")
```

Comment:
dont use ioutil.. use os.ReadFile method instead.. also please ensure the file is loaded irrespective of client working dir..

Reply:
@debanshurout

Reply:
done